INSTALL_ISSUES.md

```diff
@@ -22,3 +22,25 @@ ___
 `ImportError: liblapack.so.3: Cannot open shared object file: No such file or directory`
 
 You need to install Blas and Lapack. See [LINUX\_INSTALL.md](LINUX_INSTALL.md) for details.
+
+___
+
+`ModuleNotFoundError: No module named 'turicreate'` with Jupyter Notebook
+This is for Ubuntu>= 17.10. If you're using another distro, your command will be slightly different according to your package manager.
+
+sudo apt update
+sudo apt remove virtualenv python jupyter
+sudo apt install -y libstdc++6 python-setuptools libgconf-2-4
```

Comment:
If you are just installing turicreate, not trying to build it from source, I don't think you need these.

Reply:
It is just for safety. I didn't want to have virtualenv, jupyter through package manager if virtualenv, jupyter will be installed by pip/pip3. So I removed them.